.all-contributorsrc

```diff
@@ -131,6 +131,15 @@
       "contributions": [
         "review"
       ]
+    },
+    {
+      "login": "subagonsouth",
+      "name": "Tim Plummer",
+      "avatar_url": "https://avatars.githubusercontent.com/u/16110870?v=4",
+      "profile": "http://www.subagonsouth.com",
```

Comment:
```suggestion
      "profile": "https://github.com/subagonsouth",
```

Reply:
Good catch! I didn't realize that the bot defaulted to `www.<username>.com`. Idiotic 😅

Reply:
Oh I see now, it uses a personal website if you have one listed in your GitHub profile. That makes more sense.